src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,8 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        loger.info("getAllProductsFromShoppingCart() method successfully fetched data from DB."
```

Comment:
The log message in 'getAllProductsFromShoppingCart' should include the method name and the parameter 'userId' for clarity. For example: 'getAllProductsFromShoppingCart method successfully fetched data from DB with userId={}'.